test/test_endtoend.py

```diff
@@ -153,8 +153,9 @@ def setUp(self):
     # TODO(tyoshino): Use tearDown to kill the server.
 
     def _run_python_command(self, commandline, stdout=None, stderr=None):
+        close_fds = True if sys.platform != 'win32' else None
```

Comment:
Can you tell me why you use `None` here? The default value seems `False`.

https://docs.python.org/2.7/library/subprocess.html

Reply:
The default changed in 3.2 and 3.7. I want to use whatever the default is on Windows.

Reply:
I see, thanks.